ext/ubuntu.zsh

```diff
@@ -0,0 +1,6 @@
+#!/bin/zsh
```

Comment:
It is better to convert this into `/bin/bash`

Reply:
It is better to make the ubuntu and slackware script similar to scummvm. Just take the base URL, start, end dates. The script can then construct the date, url and download the log.

Now, we know the work involved, it might be better to write the scripts in python so that even reader.py gets fixed with the python code written for downloading of logs.